apps/heft/src/schemas/heft.schema.json

```diff
@@ -65,6 +65,83 @@
                     }
                   }
                 }
+              },
+              // Copy Files
+              {
+                "required": ["copyOperations"],
+                "properties": {
+                  "actionKind": {
+                    "type": "string",
+                    "enum": ["copyFiles"]
+                  },
+
+                  "heftEvent": {
+                    "type": "string",
+                    "enum": ["pre-compile", "compile", "bundle", "post-build"]
+                  },
+
+                  "copyOperations": {
+                    "type": "array",
+                    "description": "An array of copy operations to run perform during the specified Heft event.",
+                    "items": {
+                      "type": "object",
+                      "required": ["sourceFolder", "destinationFolders"],
+                      "properties": {
+                        "sourceFolder": {
+                          "type": "string",
+                          "description": "The folder from which files should be copied.",
+                          "pattern": "[^\\\\]"
```

Comment:
Is it okay for Heft to read from or write to files outside the project folder?  If not, should we restrict that somehow?

Reply:
I think that's fine for now. We may want to come up with a stricter set of rules at some point, though.